orttraining/orttraining/eager/ort_aten.cpp

```diff
@@ -634,6 +634,55 @@ at::Tensor slice_Tensor(
     self.options());
 }
 
+// aten::argmax.out(Tensor self, int? dim=None, bool keepdim=False, *, Tensor(a!) out) -> Tensor(a!)
+at::Tensor& argmax_out(
+const at::Tensor& self,
+c10::optional<int64_t> dim,
+bool keepdim,
+// *,
+at::Tensor& out) {
+  ORT_LOG_FN(self, dim, keepdim, out);
+
+  if (
+    !IsSupportedType(self, {at::kLong,at::kShort,at::kHalf,at::kBFloat16,at::kFloat,at::kByte,at::kInt,at::kDouble})) {
+    return at::native::call_fallback_fn<
+    &at::native::cpu_fallback,
+    ATEN_OP(argmax_out)>::call(self, dim, keepdim, out);
+  }
+  auto& invoker = GetORTInvoker(self.device());
+
+  auto ort_input_self =
```

Comment:
it seems just mapping the aten argmax call to onnx argmax op, what's the reason you can't generate it?

Reply:
Agreed. I hope to remove as many handwritten ops as possible by expanding the generator capabilities, but right now I'm working through adding some to learn/find the limitations.

Thank you for the review.